sandbox/descriptions.html.example

```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html>
```

Comment:
Could we set this up as `docs/examples/descriptions/index.html` instead of sandbox? That's the new, more appropriate place.